release/models/bgp/openconfig-bgp-bmp.yang

```diff
@@ -0,0 +1,650 @@
+module openconfig-bgp-bmp {
+
+  yang-version "1";
+
+  namespace "http://openconfig.net/yang/bgp-bmp";
+
+  prefix "oc-bgp-bmp";
+
+  import openconfig-bgp { prefix oc-bgp; }
+  import openconfig-bgp-types { prefix oc-bgp-types; }
+  import openconfig-network-instance { prefix oc-netinst; }
+  import openconfig-inet-types { prefix oc-inet; }
+  import openconfig-extensions { prefix oc-ext; }
+  import openconfig-yang-types { prefix oc-yang; }
+
+  organization
+    "OpenConfig working group";
+
+  contact
+    "OpenConfig working group
+    www.openconfig.net";
+
+  description
+    "This module defines configuration and operational state data for
+    BGP Monitoring Protocol (BMP) as defined in RFC 7854.";
+
+  oc-ext:openconfig-version "9.9.0";
```

Comment:
Is a new module proposed here (which is not a submodule) thus start the versioning at `0.1.0`